terraform_validate.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Function to run terraform init and validate in a directory
+run_terraform() {
+  local dir="$1"
+  echo "Running terraform init and validate in $dir"
+  cd "$dir" || exit
+  terraform init
+  terraform validate
+  cd - || exit
+}
+
+# Main script
+main() {
+  # Get the current directory
+  current_dir=$(pwd)
```

Comment:
This will work from package JSON but not necessarily from shell, I'd recommend using the directory the script is in as a reference point.